### DISCUSSION TITLE
Analog 4 (P29) not working on Seeed Studio XIAO RP2040

### DISCUSSION THREAD

**Discussion by eurorackworld** [shanteacontrols/OpenDeck#205] [Q&A]
I'm setting up a very simple 4 potentiometers controller on a Seeed Studio XIAO RP2040 (original, [see here](https://www.kiwi-electronics.com/nl/seeed-studio-xiao-rp2040-10969)). But only the first 3 potentiometers work (on P26, P27 and P28) but the 4th one doesn't work (on P29). Already tested if anything was wrong with the pot's and they are fine. I didn't change the RP2040 into a new one yet, but I'm assuming that wouldn't be this issue.

First I used the xiao_rp2040.uf2 file which is included in the repository here on Github. But as the 4th analog input wasn't working I created my own YML to make sure I had the rights pin selected and the config is ok. My config was quite empty, only had this:

---
  mcu: "rp2040"
  usb: true
  analog:
    type: "native"
    inputVoltage: "3.3"
    pins:
    -
      port: "0"
      index: 26
    -
      port: "0"
      index: 27
    -
      port: "0"
      index: 28
    -
      port: "0"
      index: 29

Any idea what is wrong here? Not sure if this board is 'supported officially', but this should be a quite straight out of the box working thing, not sure if I'm doing something wrong here.

**Comment by AlpesMachines** [comment]
on the official raspi2040, the 4th analog is not available on a pinheader as there is a direct connection to a temperature sensor

**Comment by paradajz** [comment]
Pin 29 is an internal pin only for temperature sensor which cannot be used as normal analog input. This was added back when I still didn't know this (and this is actually the reason I've implemented [mux-on-mux](https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant#mux-on-mux) connection scheme as I didn't have enough inputs on the OpenDeck L board). I've forgot about this target. That pin is removed now.
